Controllers/UserController.js

```diff
@@ -37,7 +76,10 @@ export const register = async (email, password, channel) => {
 }
 
 export const getMe = async (session) => {
-  console.log(session)
+  if(!session) {
```

Comment:
Also added a check for session object here. `box me` on unauthenticated user would throw an exception.

Reply:
It works and everything looks great!